shell.nix

```diff
@@ -2,16 +2,19 @@
 , defaultLv2Plugins ? false
 , lv2Plugins ? []
 , releaseMode ? false
+, enableKeyfinder ? true
```

Comment:
Alternatively we could simply have a `cFlags` parameter. What do you think?

Reply:
good idea. i added that